tox.ini

```diff
@@ -20,5 +20,5 @@ commands =
   flake8 {posargs}
 
 [flake8]
-ignore = E222,E221,F403
+ignore = E222,E221,F403,E265
```

Comment:
My main use case are "tagged comments" (an established expression may be available). In particular I've started marking python 2 compatibility code with `#PY2: …` comments, so that it's easier to find once we can finally drop that burden (probably in about 5 years :crying_cat_face:).